src/components/molecules/ProfilePictureInput/ProfilePictureInput.tsx

```diff
@@ -110,21 +128,23 @@ const ProfilePictureInput: React.FunctionComponent<PropsType> = ({
       {error && <small>Error uploading: {error}</small>}
       <small>Or pick one from our Sparkle profile pics</small>
       <div className="default-avatars-container">
-        {sparkleAvatars.map((avatar, index) => {
-          return (
-            <div
-              key={`${avatar}-${index}`}
-              className="profile-picture-preview-container"
-              onClick={() => uploadDefaultAvatar(avatar)}
-            >
-              <img
-                src={`/avatars/${avatar}`}
-                className="profile-icon profile-picture-preview"
-                alt="your profile"
-              />
-            </div>
-          );
-        })}
+        {isLoadingAvatars && <div>Loading...</div>}
+        {!isLoadingAvatars &&
+          defaultAvatars.map((avatar, index) => {
```

Comment:
Extract as a const + `useMemo`

Reply:
Changed in: https://github.com/sparkletown/sparkle/commit/6cdbdb2d81f61c67a0665f91bf358d09fc89bfbe